### ISSUE TITLE
[Feature] Show local time only if timezone of entry has a different offset

### ISSUE THREAD

**Issue by dilyanpalauzov** [TechbeeAT/jtxBoard#753]
I receive over CalDAV tasks with times (DTSTART, DUE) in UTC.  JtxBoard does correctly convert the times to my local timezone.  It shows however then both the time in UTC and in my timezone.

I do not want to see the UTC time, as it shows redundant information.

* Add an option to hide times in UTC (original times), leaving only the local time displayed.

**Comment by patrickunterwegs** [comment]
Hi @dilyanpalauzov , 
ok, I'll remove the redundant time information. 
Could you please provide a sample .ics to see how your's look like?

PS: Please always use one of the templates for new issues.

**Comment by dilyanpalauzov** [comment]
The sample is to subscribe over CalDAV using DAVx5 to server aegee.org without username, without password.  Then take any task from the OMS calendar.

**Comment by patrickunterwegs** [comment]
Note for myself:
Currently the second line with the timezone is not shown when the timezone ID is the same as the system timezone. But I stead of checking the timezone ID, the offset should be compared

**Comment by dilyanpalauzov** [comment]
With jtxBoard 2.05.03ose I still see for the aforementioned calendar twice the DUE time.  I have removed the calendar over DAVx5, the entries disappeared.  Then I added the calender back and the tasks re-appeared.

I see no option how to show in each task the time only once, converted to the local time of my device, and without reference to the timezone of the original VTODO.

**Comment by patrickunterwegs** [comment]
@dilyanpalauzov , I will reopen it for now but I just tried it and it worked. I changed the timezone of my phone to manual/London time, the second line disappeared when I killed and reopened the app. 

What timezone have you set on your phone? (If automatic, can you deactivate the automatic setting and see what appears?)

**Comment by dilyanpalauzov** [comment]
After changing time zones back and forth, it shows now only one line with the DUE time with suffix UTC.  This is still now what I want.  I want to have it in the timezone of my phone, thus without any suffix clarifying the time zone.

Is there still code to show two lines?

![Screenshot_20230823_214259_at.techbee.jtx.jpg](https://github.com/TechbeeAT/jtxBoard/assets/4992947/325e04f9-1c34-4c9e-a15d-21e54512e3fd)

**Comment by patrickunterwegs** [comment]
Sure, there are two lines if the system timezone (offset) differs from the timezone (offset) of the entry.
I don't want to show the time without the actual information that was entered. So there's why there are both lines.
What I would offer is to make the actual information with the timezone much smaller

**Comment by dilyanpalauzov** [comment]
When you receive an invitation for an online meeting, you want to see when the meeting happens in your local timezone.  You do not care in which time is the other party of the meeting, nor in which timezone the iCalendar data was transferred in machine readable time.  Do you know calendaring software, which emphasizes the timezone of the original calendar data much more, than the local timezone of the user reading the data?

Please provide an option to show all times (DUE, DTSTART) only in the local timezone of the device, hiding offsets and not spelling any timezones.

**Comment by dilyanpalauzov** [comment]
Please provide an option to show all times (DUE, DTSTART) only in the local timezone of the device, hiding offsets and not spelling any timezones.

**Comment by patrickunterwegs** [comment]
Ok, I have created a new issue for now and I will close this one as the original idea (at least my understanding) was fulfilled. 
Maybe I'm going to make a survey on Mastodon to see what other people think about this.

**Comment by dilyanpalauzov** [comment]
London has offset UTC+1.  The sample task has DUE in UTC timezone.  So London is not UTC.  I still do not understand why when I change the timezone of my device to London the DUE time is shown once, and for Central Europe timezones the due time is shown twice (with local offset and in UTC).
